lib/index.js

```diff
@@ -28,27 +30,35 @@ module.exports = function( args, options ) {
 	}
 
 	const cwd = getCwd();
-	const mainPackageName = require( path.resolve( cwd, 'package.json' ) ).name;
 	const startTime = process.hrtime();
 	const toolOptions = getOptions( options, cwd );
 	const repositoryResolver = require( toolOptions.resolverPath );
 	const forkPool = createForkPool( CHILD_PROCESS_PATH );
 
+	const mainPkgJsonPath = path.resolve( cwd, 'package.json' );
+
+	if ( !fs.existsSync( mainPkgJsonPath ) ) {
```

Comment:
The occurrence of the `package.json` file does not say "you are in the .git" repo. It can be true in 99% of cases, but I would check if the `.git/` directory is available. The program should finish if it does not exist if the `$rootRepository` is specified in the configuration.